pythran/pythonic/types/bool.hpp

```diff
@@ -3,4 +3,35 @@
 
 #include "pythonic/include/types/bool.hpp"
 
+#ifdef ENABLE_PYTHON_MODULE
+
+#include "pythonic/python/core.hpp"
+
+namespace pythonic
+{
+  template <>
+  struct to_python<bool> {
+    static PyObject *convert(bool b)
+    {
+      if (b)
+        Py_RETURN_TRUE;
+      else
+        Py_RETURN_FALSE;
+    }
+  };
+  template <>
+  struct from_python<bool> {
+    static bool is_convertible(PyObject *obj)
+    {
+      return PyBool_Check(obj);
+    }
+    static bool convert(PyObject *obj)
+    {
+      return PyInt_AsLong(obj);
+    }
+  };
+}
+
+#endif
+
```

Comment:
I agree that it work that way for now but will it be always true?
I mean, if we correctly export sliced array. Will we use the generic `to_python/from_python` or will we need the specific one for ndarray in numpy_gexpr?
I just don't want to be the one doing this job next time.